schemas/questionnaire_v1.json

```diff
@@ -76,7 +76,7 @@
             ]
           }
         },
-        "additionalProperties": false,
+        "additionalProperties": true,
```

Comment:
Instead of blanket allowing extra fields we don't know about. You should define the extra fields you are expecting.